src/storage/btree.c

```diff
@@ -2179,24 +2183,18 @@ btree_load_overflow_key (THREAD_ENTRY * thread_p, BTID_INT * btid, VPID * first_
 }
 
 /*
- * btree_delete_overflow_key () -
- *   return: NO_ERROR
- *   btid(in): B+tree index identifier
- *   page_ptr(in): Page that contains the overflow key
- *   slot_id(in): Slot that contains the overflow key
- *   node_type(in): Leaf or NonLeaf page
- *
- * Note: The overflow key is deleted. This routine will not delete the btree slot containing the key.
+ * btree_get_first_overflow_page_vpid_from_leaf - extract the vpid of the first overflow key page;
+ *                         the leaf page ptr supplied is assumed to be a leaf page with overflow
  */
 static int
-btree_delete_overflow_key (THREAD_ENTRY * thread_p, BTID_INT * btid, PAGE_PTR page_ptr, INT16 slot_id,
-			   BTREE_NODE_TYPE node_type)
+btree_get_first_overflow_page_vpid_from_leaf (THREAD_ENTRY * thread_p, const BTID_INT * btid,
```

Comment:
Function name is not accurate: "_from_leaf". It actually gets the VPID of the overflow page for both leaf and non-leaf nodes.